README.rst

```diff
@@ -38,8 +38,8 @@ Other core workflow tools
                                         pull requests.
 `python/miss-islington`_                A bot for backporting   `GitHub <https://github.com/                    `Mariatta`_
                                         CPython pull requests.  python/miss-islington/issues>`__
-`python/the-knights-who-say-ni`_        CLA enforcement bot for `GitHub <https://github.com/                    `Brett Cannon`_
-                                        Python organization     python/the-knights-who-say-ni/issues>`__
+`ambv/cla-bot`_                         CLA enforcement bot for                                                 `Łukasz Langa`_
+                                        Python organization
```

Comment:
In the "Issue tracker" column you could add the upstream link (https://github.com/edgedb/cla-bot/issues) or a link to this repo (https://github.com/python/core-workflow/issues).  @ambv, do you have any opinion on this?

Reply:
We will move ambv/cla-bot under python/cla-bot when I'm back from my July travels. I'm at EuroPython right now, are you?

Reply:
Unfortunately not.  Do you want to wait until the bot is moved to update and merge this?

Reply:
We can merge as-is and then update once it's moved.

Reply:
I merged this and created #470 as a reminder for the update.